components/ui/button/types.tsx

```diff
@@ -0,0 +1,15 @@
+export type ButtonSizes = "lg" | "md" | "sm" | "xl" | "xs";
+
+interface ButtonBaseProps {
+  /** Make the button expand the width of the container. */
+  block?: boolean;
+  /** Is the button rounded? */
+  rounded?: boolean;
+  /** The size of the button. Available sizes: `"xs" | "sm" | "md" | "lg" | "xl"` */
+  size?: ButtonSizes;
+  /** The icon component (from Heroicons) to use in the button. `icon={PlusIcon}` */
+  icon?: React.ComponentType<React.SVGProps<SVGSVGElement>>;
```

Comment:
do we have ghost button, a button for secondary action?

Reply:
We do have a `SecondaryButton` although it sill has a light background - we don't have `GhostButton` just yet, but I think I'll replace `WhiteButton` with it.

Reply:
Ah, do `WhiteButton` means a button that look like a Link ya?
That's different case.

Reply:
@mazipan No, it's [this one](https://tailwindui.com/components/application-ui/elements/buttons#component-109c4104d58d9fedfa8650dbe24c1ae8):

![image](https://user-images.githubusercontent.com/5663877/126439873-09808a11-cf72-4b12-9006-67a1aae9692a.png)

Reply:
Ah this is okay 👍